h2o-docs/src/product/productionizing.rst

```diff
@@ -210,6 +210,25 @@ Resource     Location
 Web sites    http://www.h2o.ai/steam/
 =========    ==================================================================================================
 
+Scoring Server on AWS
+~~~~~~~~~~~~~~~~~~~~~
+You can deploy a RESTful server on AWS, using the marketplace AMI (H2O Inference server - Hourly). Notice that this is a paid AMI. Transfer the
```

Comment:
Which H2O inference server?

Reply:
I couldn't find "h2o inference server" in the Amazon marketplace.

Reply:
The inference server is not in the marketplace yet. They will not publish it unless they see this documentation.